imblearn/under_sampling/_prototype_selection/tests/test_repeated_edited_nearest_neighbours.py

```diff
@@ -117,7 +118,7 @@ def test_renn_iter_wrong():
 
 
 def test_renn_fit_resample():
-    renn = RepeatedEditedNearestNeighbours()
+    renn = RepeatedEditedNearestNeighbours(n_neighbors=4)
```

Comment:
updated n_neighbours to 4 to be able to use old code with correct behaviour of the class